adapters/pulsepoint.go

```diff
@@ -0,0 +1,171 @@
+package adapters
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+
+	"github.com/prebid/openrtb"
+	"github.com/prebid/prebid-server/pbs"
+	"golang.org/x/net/context/ctxhttp"
+)
+
+type PulsePointAdapter struct {
+	http         *HTTPAdapter
+	URI          string
+	usersyncInfo *pbs.UsersyncInfo
+}
+
+// adapter name
+func (a *PulsePointAdapter) Name() string {
+	return "PulsePoint"
+}
+
+// used for cookies and such
+func (a *PulsePointAdapter) FamilyName() string {
+	return "pulsepoint"
```

Comment:
Because multiple adapters can share the same identifier (for instance the various adapters built on the appnexus platform)